android/app/src/main/java/app/organicmaps/routing/RoutingBottomMenuController.java

```diff
@@ -210,6 +216,11 @@ void showRulerInfo(@NonNull RouteMarkData[] points, Distance totalLength)
       rv.addItemDecoration(mTransitViewDecorator);
       rv.setAdapter(adapter);
       adapter.setItems(pointsToRulerSteps(points));
+
+      final ScrollView parentScroll = (ScrollView)rv.getParent();
+      if (parentScroll != null)
+        // Scroll to bottom
+        parentScroll.postDelayed(() -> parentScroll.fullScroll(ScrollView.FOCUS_DOWN), 100);
```

Comment:
nit: Add function like updateScroll() to avoid copy-paste?

Reply:
Introduced new method.